src/app/macos/Flutter/GeneratedPluginRegistrant.swift

```diff
@@ -5,6 +5,8 @@
 import FlutterMacOS
 import Foundation
 
+import shared_preferences_foundation
 
 func RegisterGeneratedPlugins(registry: FlutterPluginRegistry) {
+  SharedPreferencesPlugin.register(with: registry.registrar(forPlugin: "SharedPreferencesPlugin"))
```

Comment:
What's that?

Reply:
Was generated by the shared preferences package. Probably required.